script.js

```diff
@@ -0,0 +1,44 @@
+let saladMenuFileData = [], prices = [];
+let bill = 0, divContent="<table>";
+
+fetch("/salads.json")
+  .then((response) => response.json())
+  .then((saladMenuFileData) => createMenu(saladMenuFileData));
+
+  function setPrice(counter,saladMenuFileData){
+    if((counter + 1) % 3 == 0) return 15;
+    else if(saladMenuFileData[counter][0] == 'E' ||
+            saladMenuFileData[counter][0] == 'R' ||
+            saladMenuFileData[counter][0] == 'G' ) return 10;
+    else return 20;
+  }
+
+function createMenu(saladMenuFileData){
+  let divMenu = document.getElementById("menu");
+    for(let indexCounter=0; indexCounter < saladMenuFileData.length; indexCounter++){
+      prices[indexCounter] = setPrice(indexCounter, saladMenuFileData);
+       divContent += "<tr  >";
+       divContent += "<td class='classContentRow' id=salad" + indexCounter + ">" + (indexCounter + 1);
+       divContent += ". " + saladMenuFileData[indexCounter] +"</td>";
+       divContent += "<td> " + prices[indexCounter] + "zł</td>";
+       divContent += "<td><input type='checkbox' id='checkbox" + indexCounter + "' value='" + indexCounter + "'onclick='checkMenuBox(id,value)'></td>";
+       divContent += "</tr>";
+    }
+  divContent+="</table>";
+  divMenu.innerHTML = divContent;
+}
+
+function checkMenuBox(boxId,boxValue){
+  boxValue = parseInt(boxValue);
+  if(document.getElementById(boxId).checked == true)
+  {

```

Comment:
Raz dajesz klamerkę po enterze a raz w jednej linijce. Proponuje ustandaryzować i używać jednej notacji. Sugeruje w jedej linijce, bo jeszcze się nie spotkałem żeby ktoś dawał po enterze. Jest ku temu techniczny zarzut mianowicie w JS nie trzeba kończyć instrukcji myślnikiem ";" co oznacza, serię problemów jeżeli chcemy używać klamerek po enterze. Np.

```
const square = (numberToSquare) =>
{
  return
  {
    result: numberToSquare ** 2
  }
}
```

Oprócz tego, że wygląda to brzydko to return zwróci undefined. Dlaczego? Bo w js nie trzeba kończyć instrukcji ; i tutaj return myśli że nic nie chcemy zwrócić kończąc funkcje. True story. Możesz sprawdzić ten kod, naprawić to można przesuwając klamerkę po słówku return.